docs/source/llms/gateway/index.rst

```diff
@@ -6,7 +6,8 @@ MLflow AI Gateway (Experimental)
 
 .. warning::
 
-    The MLflow AI Gateway is a new, **experimental feature**. It is subject to modification, feature improvements, or feature removal without advance notice.
+    The MLflow AI Gateway is deprecated and will be removed in a future release.
```

Comment:
Should we remark that it's being replaced with the deployments API (just so that people don't panic :) )?

Reply:
Sounds good. I'll adjust the message.